mipengine/controller/api/error_handlers.py

```diff
@@ -16,9 +17,15 @@ class HTTPStatusCode(enum.IntEnum):
     BAD_REQUEST = 400
     BAD_USER_INPUT = 460
     INSUFFICIENT_DATA_ERROR = 461
+    ALGORITHM_EXECUTION_ERROR = 462
     UNEXPECTED_ERROR = 500
 
 
+@error_handlers.app_errorhandler(AlgorithmExecutionException)
```

Comment:
The way we integrate with the portal-backend is that we send specific status codes that will map to a specific response to the UI.

There is not need for a new "ALGORITHM_EXECUTION_ERROR" because:
1) The portal-backend won't know what to do with it,
2) The algorithm_execution_error is just an internal server error (500), it shouldn't be treated differently.

So you can just remove this and the default error handler will take care of your error and any other error that we will create in the future.

A new error handler should be added when we want a different behavior from the UI/portal-backend when that error occurs. At least, this is the logic we follow currently in the integration with MIP.

Reply:
removed it.